src/flight_computer.py

```diff
@@ -74,20 +77,25 @@ def beep(duration = 0.2):
         GPIO.output(19, GPIO.LOW)
 
 
-    def vec_len(v):
-        return np.sqrt(np.dot(v, v))
+    # def vec_len(self, v):
+    #     return np.sqrt(np.dot(v, v))
 
 
     def __standby(self):
+        start_of_acceleration_time = time.time()
         while True:
             self.rocket_data.refresh()
-            if self.vec_len(self.rocket_data.current_acceleration) < STANDBY_EXIT_THRESHOLD:
+            if (abs(self.rocket_data.current_acceleration) < STANDBY_EXIT_THRESHOLD):
```

Comment:
Does `abs()` get the magnitude?

Reply:
No i just added abs in case we got the sensor upside down